README.md

```diff
@@ -1,4 +1,4 @@
-# GitHub 从单机到联机：玩转 Pull Request
+# GitHub 从单机到联机：玩转 PULL REQUESTS
```

Comment:
Why this change? 🤣 You can write something more meaningful.

Reply:
I was just playing with Pull Requests😅, hope you don't mind.